starlette/responses.py

```diff
@@ -148,14 +150,20 @@ def delete_cookie(
         self.set_cookie(
             key,
             max_age=0,
-            expires=0,
+            expires="0",
```

Comment:
Is this needed tho? What happens if this field is not set?

Reply:
If we don't set it, `expires` is not set. I can remove it so that the semantic meaning is kept

Reply:
Although the point is to delete it. We can use the current type to achieve the same